dkms.in

```diff
@@ -644,6 +645,13 @@ read_conf()
     return $return_value
 }
 
+# Source specified variables from dkms framework configuration files.
+read_framework_conf() {
+    for i in /etc/dkms/framework.conf /etc/dkms/framework.conf.d/*.conf; do
+        [[ -e "${i}" ]] && safe_source "$i" "$@"
```

Comment:
Nit: use `$i` in the check

Reply:
Resolved in https://github.com/dell/dkms/pull/265/commits/5af2c9f4fe2686112c119ea2567b6a8b14b8738f